src/qml/filters/audio_pitch/meta.qml

```diff
@@ -0,0 +1,26 @@
+import QtQuick 2.0
+import org.shotcut.qml 1.0
+
+Metadata {
+    type: Metadata.Filter
+    isAudio: true
+    name: qsTr("Pitch")
+    mlt_service: "rbpitch"
+    qml: "ui.qml"
+    isFavorite: false
+    keyframes {
+        allowAnimateIn: false
+        allowAnimateOut: false
+        simpleProperties: ['octaveshift']
+        parameters: [
+            Parameter {
+                name: qsTr('Pitch')
+                property: 'octaveshift'
+                isSimple: true
+                isCurve: true
+                minimum: -1.0
+                maximum: 1.0
```

Comment:
This range is much smaller than `filter_rbpitch.yml`. What do you think about expanding it? -2 - 2 works with 4x - 0.25x speed without sounding too horrible IMO. 2x - 0.5x seems quite limited.

Reply:
Yes. I am open to that. I admit that I was conservative. It is easier to give more feature than to take away. Will make an updated patch shortly.